target/README.md

```diff
@@ -13,85 +13,113 @@
+-->
 
-The correlation configuration is used to specify which properties will be used to match an existing account within _{connectorName}_ to a person in _HelloID_.
+### Connection settings
 
-To properly setup the correlation:
+The following settings are required to connect to the API.
 
-1. Open the `Correlation` tab.
+| Setting  | Description                        | Mandatory |
+| -------- | ---------------------------------- | --------- |
+| UserName | The UserName to connect to the API | Yes       |
+| Password | The Password to connect to the API | Yes       |
+| BaseUrl  | The URL to the API                 | Yes       |
+
+### Correlation configuration
 
-2. Specify the following configuration:
+The correlation configuration is used to specify which properties will be used to match an existing account within _{connectorName}_ to a person in _HelloID_.
 
-    | Setting                   | Value                             |
-    | ------------------------- | --------------------------------- |
-    | Enable correlation        | `True`                            |
-    | Person correlation field  | `PersonContext.Person.ExternalId` |
-    | Account correlation field | `EmployeeNumber`                  |
+| Setting                   | Value                             |
+| ------------------------- | --------------------------------- |
+| Enable correlation        | `True`                            |
+| Person correlation field  | `PersonContext.Person.ExternalId` |
+| Account correlation field | `EmployeeNumber`                  |
 
 > [!TIP]
 > _For more information on correlation, please refer to our correlation [documentation](https://docs.helloid.com/en/provisioning/target-systems/powershell-v2-target-systems/correlation.html) pages_.
 
-#### Field mapping
+### Available lifecycle actions
+
+The following lifecycle actions are available:
+
+| Action                                  | Description                                                                                 |
+| --------------------------------------- | ------------------------------------------------------------------------------------------- |
+| create.ps1                              | Creates a new account.                                                                      |
+| delete.ps1                              | Removes an existing account or entity.                                                     |
+| disable.ps1                             | Disables an account, preventing access without permanent removal.                           |
+| enable.ps1                              | Enables an account, granting access.                                                       |
+| update.ps1                              | Updates the attributes of an account.                                                      |
+| permissions/groups/grantPermission.ps1  | Grants specific permissions to a user or group.                                            |
```

Comment:
We delen nooit rechten uit tot een groep vanuit HelloID altijd vanaf de user perspectief? Ik zou het hier alleen bij user laten.

Reply:
ok